examples/predictors/lstm/xprize_predictor.py

```diff
@@ -237,11 +235,7 @@ def _load_original_data(data_url, cutoff_date=None):
         latest_df["GeoID"] = np.where(latest_df["RegionName"].isnull(),
                                       latest_df["CountryName"],
                                       latest_df["CountryName"] + ' / ' + latest_df["RegionName"])
-        # Take a snapshot on cutoff_date
```

Comment:
No need to truncate the data frame anymore: the prediction logic will start from `start_date`, not from the last date in this data frame anymore.